Support/Testing/Travis/install.py

```diff
@@ -15,17 +15,17 @@
 dist_packages = []
 pip_packages = []
 
-linux_packages = { 'Linux-ARM':     'g++-4.8-arm-linux-gnueabihf',
-                   'Linux-X86':     'g++-5-multilib',
-                   'Linux-X86_64':  'g++-5' }
+gcc_package_versions = { 'Linux-ARM':     '-4.8-arm-linux-gnueabihf',
```

Comment:
Instead of doing this thing with package versions suffixes, I think you can just replace `g++-XXX` with `gobjc++-XXX` because the objective-C++ packages are a superset of the C++ packages.